glustercli/cmd/root.go

```diff
@@ -46,9 +39,7 @@ func init() {
 	// Global flags, applicable for all sub commands
 	RootCmd.PersistentFlags().BoolVarP(&flagXMLOutput, "xml", "", false, "XML Output")
 	RootCmd.PersistentFlags().BoolVarP(&flagJSONOutput, "json", "", false, "JSON Output")
-	RootCmd.PersistentFlags().StringVarP(&flagHostname, "glusterd-host", "", "localhost", "Glusterd Host")
-	RootCmd.PersistentFlags().BoolVarP(&flagHTTPS, "glusterd-https", "", false, "Use HTTPS while connecting to Glusterd")
-	RootCmd.PersistentFlags().IntVarP(&flagPort, "glusterd-port", "", 24007, "Glusterd Port")
+	RootCmd.PersistentFlags().StringVarP(&flagEndpoints, "endpoints", "", "http://localhost:24007", "Glusterd endpoint to connect")
```

Comment:
This should support a slice of strings.

```sh
RootCmd.PersistentFlags().StringSliceVar(&flagEndpoints, "endpoints", "", []string{"127.0.0.1:24007"}, "glusterd2 endpoints")
```

Reply:
>  glustercli doesn't support multiple endpoint URL.

That's exactly why :) We want to support that

Reply:
done